client-app/shared/catalog/components/product/properties.vue

```diff
@@ -18,15 +18,29 @@
     >
       <Vendor :vendor="product.vendor" withRating />
     </ProductProperty>
+
+    <a
+      v-if="product.properties && product.properties.length > MAX_DISPLAY_ITEMS"
+      class="flex gap-x-1 mb-4 items-center text-14 text-[color:var(--color-link)] hover:text-[color:var(--color-link-hover)] hover:cursor-pointer underline decoration-dashed"
+      @click="showAll = !showAll"
+    >
+      <VcIcon class="text-[color:var(--color-primary)]" :name="[showAll ? 'chevron-up' : 'chevron-down']" size="xs" />
+      {{
+        showAll == true ? $t("shared.catalog.product_details.see_less") : $t("shared.catalog.product_details.see_more")
+      }}
+    </a>
   </ProductTitledBlock>
 </template>
 
 <script setup lang="ts">
-import { computed, PropType } from "vue";
+import { computed, PropType, ref } from "vue";
 import _ from "lodash";
 import { Product } from "@/xapi/types";
```

Comment:
xapi inclide export * from "./types";

import { Product } from "@/xapi/types";
to
import { Product } from "@/xapi";